src/components/sidebar/Sidebar.vue

```diff
@@ -243,6 +249,14 @@ export default {
             this.hasLeaved = true
             this.timer = null
             this.isDelayOver = false
+        },
+        /**
+         * Close sidebar if close button is clicked.
+         */
+        clickedCloseButton() {
+            if (this.isFixed) {
+                if (this.isOpen && this.fullwidth) this.cancel('outside')
```

Comment:
Please put `this.cancel('outside')` on a new line underneath your last IF statement